src/panels/MissionsPanel/MissionTimeButtons.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, Group } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useAppSelector } from '@/redux/hooks';
+import { DisplayType } from '@/types/enums';
+
+import { DisplayedPhase } from './MissionContent';
+import { jumpToTime } from './util';
+
+interface MissionTimeButtonsProps {
+  isMissionOverview?: boolean;
+  currentPhase: DisplayedPhase;
+}
+
+export function MissionTimeButtons({
+  isMissionOverview,
+  currentPhase
+}: MissionTimeButtonsProps) {
+  const luaApi = useOpenSpaceApi();
+  // Since this is being used by the MissionPanel which already subscribes to time, we
+  // and this component re-renders every second due to parent being re-rendered,
+  // we don't need to do any subscription here- TODO: do we want to anyways? obs this will be
+  // one subscription for every button we have...
+  const now = useAppSelector((state) => state.time.timeCapped);
```

Comment:
I looked at the function for where this is used and it contains a lot of logic. It seems a bit "magic" that we interpolate if the time difference is less than a day, or jump otherwise. I wonder if there was some specific motivation behind that, and if the interpolation case is really necessary. 

Either way, maybe we could make a lua function that does this logic instead, including the fading?  That is, make a `openspace.time.jumpToTime` function, that works similarly to `openspace.pathnavigation.jumpTo`, `openspace.globebrowsing.jumpToGeo` etc.. That could be useful in more situations, and it would simplify the logic as well as get rid of the dependency of the current time.

Reply:
It would also give the user control over how long the fading takes, and make it consistent with the other transitions :) 

So I'd say, let's do it!